assignments/function-conversion.js

```diff
@@ -3,22 +3,30 @@
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
-// myFunction();
+let myFunction = () => {console.log("Function was invoked!");}
+
+myFunction();
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
-// anotherFunction("Example");
+let anotherFunction = param => {return param;}
```

Comment:
```suggestion
let anotherFunction = param => param;
```
The way you have it is fine if that's what's easier for you to read, I'm just showing you here that it works this way too. The arrow is an implicit return statement.